theano/gpuarray/tests/test_rng_mrg.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import, print_function, division
 import functools
 
-import numpy
+import numpy as np
```

Comment:
Don't mix that in other PR! Don't loose time to remove it, but don't do it again...